### ISSUE TITLE
About the game operation.

### ISSUE THREAD

**Issue by giglf** [Zaphyk/project-skylines#9]
Hello, I am just play your game. It's wonderful!
The goal of the game is achieving higher point. 
But have you found that, if I just press one key at first, like 'a'. The plane will flying in a circle without crash into the world. And finally I can get as higher as score as I want.
Can this be improved? :-)

**Comment by Zaphyk** [comment]
Yes, this is one of the main issues stopping me from creating a global leaderboard. I am still thinking how to solve the issue without affecting the gameplay too much. I am open to ideas

**Comment by giglf** [comment]
I have tried several times to test the another bug that the score display finally have only 3 digits. 
But then I found something.
Sometimes if I am just holding down one key and it will crash around 900-1100 scores. 
The flying circle will be bigger when the speed higher. 
How about add a detection when holding one key too long like 3 or 5 sec, the speed will sudden increase. Or just let the plane without control?
But I think it's a bad solution. 
However, I don't have any ideas when I think it's hard to modify the world to avoid it.
Hope it can help you.
